app/controllers/spree/page_not_found_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Spree
```

Comment:
Please change the controller name to 'StaticLinks' and add page_not_found action in it. As we need a controller for footer links. So we can use this controller for footer actions.